### PR TITLE
Fix Categories serializing

### PR DESCRIPTION
**Pull request by tcitworld**
Commas are not to be escaped

Will require #28 for Travis fixes

### CODE REVIEW

**Comment by lpil** [comment]
Could you rebase on master please? :)

File: lib/icalendar/util/kv.ex [hunk 1]
```diff
@@ -37,7 +37,7 @@ defmodule ICalendar.Util.KV do
   end
 
   def build("CATEGORIES" = key, value) do
-    build_sanitized(key, Enum.join(Value.to_ics(value), ","))
+    "#{key}:#{Enum.join(Value.to_ics(value), ",")}\n"
```
Comment (lpil) [inline lib/icalendar/util/kv.ex]: Can write "CATEGORIES" here rather than interpolating

Comment (tcitworld) [reply lib/icalendar/util/kv.ex]: Indeed, missed that.

**Comment by tcitworld** [comment]
Fixed and rebased.

**Review by lpil — APPROVED** [review]
Thank you

**Comment by tcitworld** [comment]
@lpil Thanks! Do you mind creating a new release ?

**Comment by lpil** [comment]
I believe I have https://hex.pm/packages/icalendar/0.7.1

Could you check that for me please? Thank you

**Comment by tcitworld** [comment]
Very sorry, I forgot I added my github fork as the dependency, and therefore no update would show up. ><
Thanks !

**Comment by lpil** [comment]
Fab! Glad it's working

**Pull request merged**